chemprop/args.py

```diff
@@ -229,6 +233,10 @@ class TrainArgs(CommonArgs):
     """
     ignore_columns: List[str] = None
     """Name of the columns to ignore when :code:`target_columns` is not provided."""
+    atom_targets: List[str] = []
+    """Name of the columns containing target atomic values."""
+    bond_targets: List[str] = []
+    """Name of the columns containing target bond values."""
```

Comment:
If these two were left blank, any way we could figure out which targets belonged in which columns? Like under normal molecule-level operation you would just run `--get_task_names` and assume that they were all target columns by default. Is there a data utils function that could be addressed to make these default more easily?

Reply:
If they were left blank, this model would not do anything. I would build a data utils function to address this. Really appreciate this suggestion.

Reply:
If you think there's a function that can determine which atom/bonds are which and it works really well, then you wouldn't even need the atom_feature_targets and bond_feature_targets arguments. You could remove them and use just target_columns to say which of those to use and which to ignore (using all by default). If choosing which ones are atom and which are bond is hard or unreliable though, then the current set up would be better.

Reply:
That is a good idea! I added a `get_mixed_task_names()` function in data/utils.py, `--atom_targets` and `--bond_targets` have been removed and `--target_columns` becomes the default option.